src/nvim/eval.c

```diff
@@ -10778,44 +10778,78 @@ static void f_jobresize(typval_T *argvars, typval_T *rettv)
   rettv->vval.v_number = 1;
 }
 
-// "jobstart()" function
-static void f_jobstart(typval_T *argvars, typval_T *rettv)
+static char **tv_to_argv(typval_T *cmd_tv, char **cmd)
 {
-  rettv->v_type = VAR_NUMBER;
-  rettv->vval.v_number = 0;
+  if (cmd_tv->v_type == VAR_STRING) {
+    char *cmd_str = (char *)get_tv_string(cmd_tv);
+    if (cmd) {
```

Comment:
:+1: -- This function has many `NULL` comparisons. I decided to just change the explicit tests to implicite.